_posts/2020-06-12-coding-with-kids.md

```diff
@@ -0,0 +1,195 @@
+
+```python
+subjects = ["Dad","Mother","Kid1","Kid2"]
+deeds = ["eats","kicks","burns","lifts"]
+objects = ["lion","bicycle","airplane","ice cream"]
+
+from random import randint
+def pickword(words):
+    wordcount = len(words)
+    pickedword = randint(0, wordcount-1)
+    return words[pickedword]
+
+print(pickword(subjects),pickword(deeds),pickword(objects),end='.\n')
+```
+
+This will lead into phrases like `Dad eats lion` and `Mother kicks airplane` or the sinful deed `Dad burns ice cream`. 
+
+#### RPG with Python
+
+Quickly our learning how to implement software with Python turned into how to control characters in role playing game with Python. We have been using [Code Combat](https://codecombat.com/) for that. Beside Python, it also supports JavaScript. Kids will quite quickly get grip on the game, but there are missing translations and some new features that will require a parent to support the kid on the way. Code Combat is implemented in a way that supports problem solving, learning algorithms and learning basics of object oriented programming. It is implemented with kind of Domain Specific Language. You can only use given commands and equipment will give your character more options. For example your hero can't cleave with sword without a sword. 
+
+We also tried once to go further and check if we could find something from [CodinGame](https://www.codingame.com/) that we could do. Unfortunately it seems that the challenges there were still a bit too hard. Maybe we get back there in few years. 
+
+
+## Creating the first home page - the hard way
+
+Last and maybe the least is making the home pages. I was once told that the easiest way to make home pages is to make a word document and then save it as a HTML file. If you know anything about home pages, you might know that there are plenty of problems that will arise from that approach. Instead, I taught my kids the same painful route that I took 25 years ago. Basically we opened up a notepad and then into another screen the [w3schools](https://www.w3schools.com/html/default.asp). Then we made an index document using the simplest possible syntax with having title with two words and the body with a sentence. We hosted it in Azure Blob Storage with static websites feature enabled that allows you to serve static html files from the blog storage. I had to check out what it has costed me, and I figured out that less than 1c. Seems like a cheap way to start. 
```

Comment:
Basically, (comma)

having a title

HTML

It seems like a ?

Needless to say, (comma)

was a kind of